charters/MetadataSchema/charter.md

```diff
@@ -0,0 +1,63 @@
+
+# Metadata Schema
```

Comment:
nit: If this document describes `Metadata Schema Committee`, then that should probably be the title and directory name.

Reply:
We probably need to have a broader discussion about style of how we refer to the charters etc in these documents so we can all be consistent across the different documents but for now I have removed the word committee.